hpms/pom.xml

```diff
@@ -100,5 +100,4 @@
             </plugin>
         </plugins>
     </build>
-
```

Comment:
Whitespace changes in an unrelated module should be avoided, especially with a PR this big.

Reply:
ok